index.bs

```diff
@@ -779,6 +782,16 @@ spec:dom-ls; type:interface; text:Document
     dangerous and should not be used if you have Real Deal auth and csrf cookies
     used.
 
+   * '<dfn>`unsafe-include-credentials`</dfn>' All cross-origin requests will
```

Comment:
Good point. I'll rewrite it so it only applies to requests to the physical origin.

For simplicity, I think setting it to false should be ignored (much simpler for implementation). Will clarify.